activejob/lib/active_job/serializers/time_with_zone_serializer.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ActiveJob
+  module Serializers
+    class TimeWithZoneSerializer < ObjectSerializer # :nodoc:
+      def serialize(time)
+        super("value" => time.iso8601)
+      end
+
+      def deserialize(hash)
+        Time.iso8601(hash["value"]).in_time_zone
```

Comment:
If we need this serializer we are missing a test.

Reply:
I've just added test.